ResearchKit/Consent/ORKVisualConsentTransitionAnimator.m

```diff
@@ -47,7 +47,7 @@ @interface ORKVisualConsentAnimationContext : NSObject
 @property (nonatomic, copy) ORKVisualConsentAnimationCompletionHandler handler;
 @property (nonatomic, copy) ORKVisualConsentAnimationCompletionHandler loadHandler;
 
-@property (nonatomic, strong) NSValue *startTime;
+@property (nonatomic, copy) NSValue *startTime;
```

Comment:
This was more about consistency, to just use `copy` as the default for things that conform to `NSCopying`.